src/elm/Page/Profile/Editor.elm

```diff
@@ -0,0 +1,476 @@
+module Page.Profile.Editor exposing (Model, Msg, init, msgToString, update, view)
+
+import Api
+import Api.Graphql
+import Avatar exposing (Avatar)
+import File exposing (File)
+import Graphql.Http
+import Html exposing (Html, button, div, form, input, label, span, text, textarea)
+import Html.Attributes exposing (accept, class, disabled, for, id, multiple, style, type_, value)
+import Html.Events exposing (onInput)
+import Http
+import I18Next exposing (Translations, t)
+import Icons
+import Json.Decode
+import Page
+import Profile exposing (Profile)
+import Route
+import Session.LoggedIn exposing (External(..), FeedbackStatus(..))
+import Session.Shared exposing (Shared)
+import UpdateResult as UR
+
+
+
+-- INIT
+
+
+init : Session.LoggedIn.Model -> ( Model, Cmd Msg )
+init loggedIn =
+    let
+        profileQuery =
+            Api.Graphql.query loggedIn.shared
+                (Profile.query loggedIn.accountName)
+                CompletedProfileLoad
+    in
+    ( initModel
+    , profileQuery
+    )
+
+
+
+-- MODEL
+
+
+type alias Model =
+    { fullName : String
+    , email : String
+    , bio : String
+    , location : String
+    , interests : List String
+    , interest : String
+    , status : Status
+    , avatar : Maybe Avatar
+    , wasSaved : Bool
+    }
+
+
+initModel : Model
+initModel =
+    { fullName = ""
+    , email = ""
+    , bio = ""
+    , location = ""
+    , interests = []
+    , interest = ""
+    , status = Loading
+    , avatar = Nothing
+    , wasSaved = False
+    }
+
+
+type Status
+    = Loading
+    | LoadingFailed (Graphql.Http.Error (Maybe Profile))
+    | Loaded Profile
+
+
+
+-- VIEW
+
+
+view : Session.LoggedIn.Model -> Model -> { title : String, content : Html Msg }
+view loggedIn model =
+    let
+        title =
+            t loggedIn.shared.translations "profile.edit.title"
+
+        content =
+            case model.status of
+                Loading ->
+                    Page.fullPageLoading
+
+                LoadingFailed _ ->
+                    Page.fullPageError (t loggedIn.shared.translations "profile.title") Http.Timeout
+
+                Loaded profile ->
+                    view_ loggedIn model profile
+    in
+    { title = title
+    , content = content
+    }
+
+
+view_ : Session.LoggedIn.Model -> Model -> Profile -> Html Msg
+view_ loggedIn model profile =
```

Comment:
something is off here. checkout how the font size looks bigger:

<img width="1339" alt="Screen Shot 2020-07-15 at 14 33 07" src="https://user-images.githubusercontent.com/1082127/87576678-1bb6ed80-c6a8-11ea-838c-7027d13ca5d1.png">
<img width="845" alt="Screen Shot 2020-07-15 at 14 33 02" src="https://user-images.githubusercontent.com/1082127/87576684-1ce81a80-c6a8-11ea-9320-f4fa628120c6.png">

Reply:
Good catch! I rechecked the [design in Figma](https://www.figma.com/file/oZJUIyHRmLyLIKDxYebvLg/Form-elements?node-id=0%3A1) for the form elements and improved CSS rules a bit.